src/foam/nanos/session/Session.js

```diff
@@ -136,6 +136,11 @@ foam.CLASS({
       javaFactory: 'return reset(getX());',
       hidden: true,
       transient: true
+    },
```

Comment:
Let's not add this direction to the Session, otherwise we would be adding stuff to the session all of the time and it would be specific to our use case rather than generic. Instead, the Session has a Context, which is where the value can be added generically. My mistake. When I say to "add something to the Session", what I really mean is "add to the Session Context". Otherwise, everything looks good.

Reply:
do you mind me using just a "homeDenomination" string?